vnext/Microsoft.ReactNative/ReactInstanceSettings.cpp

```diff
@@ -7,4 +7,11 @@
 #include "ReactInstanceSettings.g.cpp"
 #endif
 
-namespace winrt::Microsoft::ReactNative::implementation {} // namespace winrt::Microsoft::ReactNative::implementation
+namespace winrt::Microsoft::ReactNative::implementation {
+
+/*static*/ ReactPropertyBag ReactInstanceSettings::GetGlobalProperties() noexcept {
+  static ReactPropertyBag s_globalProperties;
```

Comment:
>s_globalProperties [](start = 26, length = 18)

[I haven't seen the part of the code that might make this clear, writing this comment just so that I don't forget the question] 
What makes a default-constructed property bag the global property bag?

Reply:
We just declare a variable, and then we use it to return the GlobalProperties() value.

Reply:
The global properties are removed until someone has a request for them.